src/Inventory/Conf.php

```diff
@@ -149,8 +125,8 @@ public function knownInventoryExtensions(): array
      */
     public function importFile($files): Request
     {
-        $path = $files['importfile']['tmp_name'];
-        $name = $files['importfile']['name'];
+        $path = $files['inventory_file']['tmp_name'];
```

Comment:
Changing this field name will produce a BC-break, even if the probability that a plugin will use this method is really limited.
Is this really needed ?